index.html

```diff
@@ -0,0 +1,221 @@
+        <p class="play-role">Back End Dev</p>
+        <img class="cunter" src="/images/Counter.png" alt="counter" />
+        <p class="year">2015</p>
+    </div>
+     <p class="project-description">
+        A daily selection of privately personalized reads; no accounts or
+        sign-ups required.
+     </p>
+     <ul class="p-languages">
+        <li class="language">html</li>
+        <li class="language">css</li>
+        <li class="language">java script</li>
+     </ul>
+     <button class="button" type="button">See Project</button>
+    </div>
+    </div>
+    <div class="work-border-two">
+        <img class="portfolio-img" src="/images/Snapshoot Portfolio4.png" alt="portfolio image" />
+    <div class="portfolio-section">
+        <h3 class="project-name">Tonic</h3>
+    <div class="project-details">
+        <h4 class="project-owner">CANOPY</h4>
+        <img class="cunter" src="/images/Counter.png" alt="counter" />
+        <p class="play-role">Back End Dev</p>
+        <img class="cunter" src="/images/Counter.png" alt="counter" />
+        <p class="year">2015</p>
+    </div>
+     <p class="project-description">
+        A daily selection of privately personalized reads; no accounts or
+        sign-ups required.
+     </p>
+     <ul class="p-languages">
+        <li class="language">html</li>
+        <li class="language">css</li>
+        <li class="language">java script</li>
+     </ul>
+      <button class="button" type="button">See Project</button>
+    </div>
+    </div>
+   </section>
+
+   <section class="about-myself">
+    <div class="about-details">
+        <h2 class="about-header">About Myself</h2>
+        <p class="about-content">Hello! I’m a software developer! I can help you build a product ,
+            features or website. Look through some of my work and projects! If you
+            like what you see and have a project you need to be coded, don’t
+            hestiate to contact me.
+        </p>
+        <h3 class="social-title">LET'S CONNECT</h3>
+        <ul class="social-links">
+            <li class="social-link">
+                <a href="#"><img class="social-icon" src="/images/twitter.svg" alt="Twiter logo"/></a>
+            </li>
+            <li class="social-link">
+                <a href="#"><img class="social-icon" src="/images/Linkedin icon.svg" alt="Linkdin logo"/></a>
+            </li>
+            <li class="social-link">
+                <a href="#"><img class="social-icon" src="/images/medium.svg" alt="Medium logo" /></a>
+            </li>
+            <li class="social-link">
+                <a href="#"><img class="social-icon" src="/images/github.svg" alt="Github Logo" /></a>
+            </li>
+            <li class="social-link">
+                <a href="#"><img class="social-icon" src="/images/angellist.svg" alt="Angellist" /></a>
+            </li>
+        </ul>
+        <button class="resume-buton" type="button">Get My Resume</button>
```

Comment:
- [ ] Please 🙏🏽 use the lower letters in the corresponding words to be similar to the Figma design.

| Figma design | Your design |
|---|---|
| ![image](https://github.com/ariful04/Myportfolio/assets/58642949/cbc308ac-ce34-4976-a89b-f2846f89e330) | ![image](https://github.com/ariful04/Myportfolio/assets/58642949/44dbdb1d-a634-498b-84e6-d23ab33ff66a) |